src/main/java/frc/robot/commands/ShooterCommand.java

```diff
@@ -30,6 +32,7 @@ public ShooterCommand(ShooterSubsystem shooter) {
     // Called when the command is initially scheduled.
     @Override
     public void initialize() {
+        Constants.m_RobotState.setState(States.SHOOTING);
```

Comment:
Shooting command is always on because it only turns the wheel on - The shooting state should be set when the belt is moving the balls up in order to shoot.